examples/features/errors/client/main.go

```diff
@@ -30,9 +32,14 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+var port = flag.Int("port", 50052, "port number")
```

Comment:
Let's make the client take address, not port.
https://github.com/grpc/grpc-go/blob/master/examples/features/cancellation/client/main.go#L35